### DISCUSSION TITLE
how to create github stats like u

### DISCUSSION THREAD

**Discussion by devsubid** [JasonLovesDoggo/JasonLovesDoggo#7] [General]
how to create github stats like u?
u used a svg but how to get it live and automatic?

**Comment by JasonLovesDoggo** [comment]
> how to create github stats like u?
> u used a svg but how to get it live and automatic?

Which stat because I use a bunch of different stuff

**Reply by JasonLovesDoggo** [reply]
If you want you can look at the raw README.md file to see the websites

**Comment by devsubid** [comment]
![image](https://user-images.githubusercontent.com/85873704/185289733-261eb819-7077-4aef-9145-ca9beb391cc0.png)
this one

**Reply by JasonLovesDoggo** [reply]
I will answer that tomorrow once I get to my PC

**Reply by devsubid** [reply]
ok, I will be there to get ur help. Thanks for ur response.

**Reply by JasonLovesDoggo** [reply]
> ok, I will be there to get ur help. Thanks for ur response.

WELP, i use  something called https://metrics.lecoq.io/ 

so sorry for forgetting
